sphinxcontrib/images.py

```diff
@@ -252,20 +255,20 @@ def download_images(app, env):
             'Downloading remote images...',
             brown,
             len(env.remote_images)):
-                
+
         dst = os.path.join(env.srcdir, env.remote_images[src])
         if not os.path.isfile(dst):
-            app.info('{} -> {} (downloading)'
+            logger.info('{} -> {} (downloading)'
```

Comment:
that's how logging was done in sphinx, I'm not up to date, but in sphinx2 should that be changed to logging? is moving to logging compatible in sphinx1?

Reply:
```
from sphinx.util import logging
logger = logging.getLogger(__name__)
```

can be found in many files of the sphinx project, for both 1.6, 1.7, 1.8 and 2.x tags, e.g.:

Not here: https://github.com/sphinx-doc/sphinx/blob/v1.5.6/sphinx/application.py
But here: https://github.com/sphinx-doc/sphinx/blob/v1.6/sphinx/application.py
And here: https://github.com/sphinx-doc/sphinx/blob/v2.1.2/sphinx/application.py

So it seems compatible for 1.6 or newer if I am any judge.